src/content/configuration/node.md

```diff
@@ -10,7 +10,7 @@ contributors:
   - EugeneHlushko
 ---
 
-These options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
+The following node options below configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
```

Comment:
I think we reference node officially as Node.js throughout the docs, lets do that here too if you dont mind
```suggestion
The following Node.js options below configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
```